modules/programs/aerc-accounts.nix

```diff
@@ -177,21 +177,20 @@ in {
               params = cfg.aerc.smtpOauth2Params;
             };
 
-            protocol = if smtp.tls.enable && !smtp.tls.useStartTls then
-              "smtps${loginMethod'}"
+            protocol = if smtp.tls.enable then
+              if smtp.tls.useStartTls then
+                "smtp${loginMethod'}"
+              else
+                "smtps${loginMethod'}"
             else
-              "smtp${loginMethod'}";
+              "smtp${loginMethod'}+insecure";
```

Comment:
not sure if it actually makes a difference, but i would probably swap the order here for clarity still, because `smtp+insecure` corresponds to the `scheme` and `plain` corresponds to the `auth` method (separated by `+`)

appart from that, this LGTM!